src/components/ResourceBrowser/ResourceList/K8SResourceList.tsx

```diff
@@ -318,7 +400,7 @@ export const K8SResourceList = ({
     }
 
     const renderList = (): JSX.Element => {
-        if (filteredResourceList.length === 0) {
+        if (filteredResourceList?.length === 0 || resourceListDataError) {
```

Comment:
In case of resourceListDataError there should be some error page

Reply:
I am showing a toast on error for now. When we use resource filter (the new demo feature) it throws an error if no resources match the filter conditions. So this can be a point of discussion for the product team otherwise I feel we should let it stay as is for now.

Ideally instead of throwing toasts we should show the error using ErrorScreenManager.